packages/auth/src/index.ts

```diff
@@ -102,9 +102,17 @@ export function initIocContainer(
     })
   })
 
-  container.singleton('openApi', async (deps) => {
-    const config = await deps.use('config')
-    return await createOpenAPI(config.authServerSpec)
+  container.singleton('openApi', async () => {
+    const authServerSpec = await createOpenAPI('./openapi/auth-server.yaml')
+    const resourceServerSpec = await createOpenAPI(
+      './openapi/resource-server.yaml'
+    )
+    const idpSpec = await createOpenAPI('./openapi/id-provider.yaml')
```

Comment:
`auth` build script needs to add these to the `dist` folder:
https://github.com/interledger/rafiki/blob/889807585763faaf4fc68d9a5d49140776d12501/packages/auth/package.json#L11
See `backend`'s `copy-files`:
https://github.com/interledger/rafiki/blob/889807585763faaf4fc68d9a5d49140776d12501/packages/backend/package.json#L8-L10
(this is causing local env to fail even after running `fetch-schemas`)